packages/core/src/connectors/aliyun-dm/index.ts

```diff
@@ -24,19 +26,32 @@ export const metadata: ConnectorMetadata = {
   },
 };
 
-interface Template {
-  type: keyof EmailMessageTypes;
-  subject: string;
-  content: string; // With variable {{code}}, support HTML
-}
+export const validateConfig: ValidateConfig = async (config: unknown) => {
+  if (!config) {
+    throw new ConnectorConfigError('Missing config');
+  }
+
+  const result = configGuard.safeParse(config);
+  if (!result.success) {
+    throw new ConnectorConfigError(result.error.message);
+  }
+};
```

Comment:
will it be better to wrap `ConnectorConfigError` simply with an error code and a zod error object so we can see the structured error data? use `result.error.message` will likely lose context. also the available init param of `ConnectorConfigError` should be a const / enum just like `RequestError`.

Reply:
I was tring to keep the connector independant with other Logto's code, so that the connector contributor can move on quickly.
So I think is no need to introduce Logto's error code. Created LOG-1381 for further discussion.